apps/web/lib/metadata.ts

```diff
@@ -41,7 +41,13 @@ export const prepareRootMetadata = (recipe: RootMetadataRecipe): Metadata => ({
     ],
   },
   manifest: "/site.webmanifest",
-  viewport: "width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0",
+  viewport: {
```

Comment:
PageWrapper (used for pages router routes) has
```html
      <Head>
        <meta
          name="viewport"
          content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0, viewport-fit=cover"
        />
      </Head>
```

and the equivalent in App router is this viewport metadata. We need to use the object syntax.